corrscope/renderer.py

```diff
@@ -595,6 +607,11 @@ def add_lines_stereo(
                 chan_line: Line2D = ax.plot(
                     xs, chan_zeros, color=line_color, linewidth=line_width
                 )[0]
+                if cfg.line_outline_width:
```

Comment:
It's a good thing that you don't render an outline if transparent. Without this if statement, you might get a speed regression.